### ISSUE TITLE
--prebuilt-only for all commands: only substitute, don't build

### ISSUE THREAD

**Issue by domenkozar** [NixOS/nix#3691]
Currently only `nix-env` supports `--prebuilt-only` (also not the best name), it would be great if all commands would do that.

Probably something like `--substitute-only`. Currently there's a way to pass empty builders, but I was today on ZuriHac and most people didn't think of that.

**Comment by matthewbauer** [comment]
Yeah i think you can do `--builders '' --max-jobs 0` to do this. But a more memorable flag would be nice.

**Comment by domenkozar** [comment]
Is there a way to increase number of jobs and substitute only?

I'd like to do:

   nix-build -j 10 --substitute-only || true
   nix-build -j2

As a workaround.
